Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -1691,3 +1704,9 @@ std::pair<double, double> HGCalGeomParameters::cellPosition(
 void HGCalGeomParameters::rescale(std::vector<double>& v, const double s) {
   std::for_each(v.begin(), v.end(), [s](double& n) { n *= s; });
 }
+
+void HGCalGeomParameters::resetZero(std::vector<double>& v) {
+  for (auto n : v)
+    if (std::abs(n) < 1.e-20)
```

Comment:
magic number `1.e-20` should be named constant

Reply:
Done